pkg/systemmeta/labels.go

```diff
@@ -0,0 +1,13 @@
+package systemmeta
+
+// Built-in labels.
+const (
+	// ResourceTypeLabel is the label key to indicate the system type of delegated resource.
+	ResourceTypeLabel = "resource.walrus.seal.io/type"
+	// ConnectorTypeLabel indicates the connector type of the resource.
+	ConnectorTypeLabel string = "connector.walrus.seal.io/type"
```

Comment:
"walrus.seal.io/connector" = "type"

Reply:
This pattern is not common. Let's use dot for hierarchical labels, e.g., https://kubernetes.io/docs/reference/labels-annotations-taints/#kubernetes-io-config-hash